sdk/src/main/java/com/bugsnag/android/User.java

```diff
@@ -1,28 +1,23 @@
 package com.bugsnag.android;
 
+import android.content.SharedPreferences;
 import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 
 import java.io.IOException;
 
 /**
  * Information about the current user of your application.
  */
 class User implements JsonStream.Streamable {
```

Comment:
This class would need to be public if exposed as part of the API

Reply:
Of course!